### PR TITLE
Add cones as glyphs

### PR DESCRIPTION
**Pull request by GuillaumeFavelier**
As suggested by @agramfort this PR adds a function `cone()` that creates cone glyphs given `centers`, `resolutions` or `heights` with some basic testing in `fury/tests/test_actors.py::test_cones`.

image

### CODE REVIEW

**Comment by Garyfallidis** [comment]
Looks good will review asap.

File: fury/actor.py [hunk 1]
```diff
@@ -1451,6 +1451,99 @@ def sphere(centers, colors, radii=1., theta=16, phi=16,
...
+    """Visualize one or many cones with different colors and radii
+
+    Parameters
+    ----------
+    centers : ndarray, shape (N, 3)
+    colors : ndarray (N,3) or (N, 4) or tuple (3,) or tuple (4,)
+        RGB or RGBA (for opacity) R, G, B and A should be at the range [0, 1]
+    radii : float or ndarray, shape (N,)
+    theta : int
+    phi : int
```
Comment (skoudoro) [inline fury/actor.py]: Can you update the documentation? It missed `directions`, `height`, `resolutions` and some parameters needs to be removed.

File: fury/actor.py [hunk 2]
```diff
@@ -1451,6 +1451,99 @@ def sphere(centers, colors, radii=1., theta=16, phi=16,
...
+    >>> scene = window.Scene()
+    >>> centers = np.random.rand(5, 3)
+    >>> directions = np.random.rand(5, 3)
+    >>> heights = np.random.rand(5)
+    >>> cone_actor = actor.cone(centers, directions, (1, 1, 1), heights)
+    >>> scene.add(cone_actor)
+    >>> # window.show(scene)
+
+    """
+
```
Comment (skoudoro) [inline fury/actor.py]: Docstyle: can you remove this empty line :smile:

**Comment by skoudoro** [comment]
Great, thank for the update! 

Any opinions @Garyfallidis ? I will wait until tomorrow morning to merge it

**Comment by Garyfallidis** [comment]
It's good. Go ahead @skoudoro and merge! Thank you @GuillaumeFavelier.

**Pull request merged**